app/scripts/lib/local-store.js

```diff
@@ -70,7 +74,28 @@ export default class ExtensionStore {
   _set(obj) {
     const { local } = extension.storage
     return new Promise((resolve, reject) => {
-      local.set(obj, () => {
+      const state = cloneDeepWith(obj, (value, key) => {
+        // On Firefox, native Errors cannot be cloned
+        if (value instanceof Error) {
+          if (!stateErrorFound) {
+            stateErrorFound = true
+            const error = new Error(`Native error found at key '${key}'`)
+            captureException(error, {
+              extra: {
+                originalError: value,
+              },
+            })
+          }
+          return {
```

Comment:
I could probably replace this with `serialize-error` or something similar, but there were a few bugs on the `serialize-error` GitHub repo that might affect us. This seemed effective enough for now :shrug: